plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/gettingstarted/GettingStartedAuthUtils.kt

```diff
@@ -229,13 +233,27 @@ fun reauthenticateWithQ(project: Project) {
 }
 
 fun emitUserState(project: Project) {
-    AuthTelemetry.userState(
-        project,
-        source = getStartupState().toString(),
-        authEnabledConnections = getEnabledConnections(project),
-        authStatus = getAuthStatus(project),
-        passive = true
-    )
+
+    val scopes = ToolkitAuthManager.getInstance().listConnections().flatMap { connection ->
+        when (connection) {
+            is ProfileSsoManagedBearerSsoConnection -> connection.scopes
+            is LegacyManagedBearerSsoConnection -> connection.scopes
+            else -> emptyList()
+        }
+    }.toSet()
+
+    TelemetryService.getInstance().record(project) {
+        datum("auth_userState"){
+            createTime(Instant.now())
+            unit(Unit.NONE)
+            value(1.0)
```

Comment:
these are implicit

Reply:
This metric is currently in telemetryOverride. We need to use the one from common.

Reply:
I understand that our AuthTelemetry is being generated. no version of AuthTelemetry.userState() accepts authScopes as a parameter. I think this might be why? :

https://github.com/aws/aws-toolkit-common/blob/49de773b65efdeaadea5c30859cf286ccd74ec5b/telemetry/definitions/commonDefinitions.json#L2708C1-L2710C19

I'll try making some changes to common locally to see if I can generate a version of the function that does.